components/PearlPage/PearlMetrics.jsx

```diff
@@ -0,0 +1,87 @@
+import { get7DaysAvgActivity } from 'common-util/api/flipside';
+import { SUB_HEADER_CLASS } from 'common-util/classes';
+import { PREDICTION_ECONOMY_DASHBOARD_URL } from 'common-util/constants';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card } from 'components/ui/card';
+import { ExternalLink } from 'components/ui/typography';
+import { usePersistentSWR } from 'hooks';
+import Image from 'next/image';
+import { useMemo } from 'react';
+
+const fetchMetrics = async () => {
+  const [dailyActiveAgents] = await get7DaysAvgActivity();
+
+  return {
+    dailyActiveAgents:
+      dailyActiveAgents.status === 'fulfilled' ? dailyActiveAgents.value : null,
```

Comment:
there's no need to check the status if it's not from promise.allSettled. it's not working now :( 
<img width="526" alt="image" src="https://github.com/user-attachments/assets/bbb6ab5e-104a-4f97-bec0-f0480668becb" />